orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -361,7 +361,7 @@ def __init__(self, master):
         range_widget = widgets.RangeWidget(None, self, ('min_df', 'max_df'),
                                            minimum=0., maximum=1., step=0.05,
                                            allow_absolute=True)
-        range_widget.setToolTip(preprocess.FrequencyFilter.tooltip)
+        range_widget.setToolTip(preprocess.FrequencyFilter.__doc__)
```

Comment:
Could we get rid of this empty line?
<img width="576" alt="screen shot 2016-08-12 at 18 16 31" src="https://cloud.githubusercontent.com/assets/713026/17629096/f563f5aa-60b8-11e6-896e-3a7fcb336f8e.png">